db/db.js

```diff
@@ -209,19 +222,19 @@ async function getMostRecentSessionPhone(twilioPhone){
     }
```

Comment:
Should `getMostRecentSession`, `getSessionWithSessionId`, and `getMostRecentSessionPhone` also use qcreateSessionFromRow`?

Reply:
I only changed things that are necessary for using the object that the template needs, so not sure

Reply:
Fair enough :+1: